Formula/g/gnome-autoar.rb

```diff
@@ -26,10 +26,21 @@ class GnomeAutoar < Formula
 
   depends_on "meson" => :build
   depends_on "ninja" => :build
-  depends_on "pkg-config" => :build
+  depends_on "pkg-config" => [:build, :test]
+
+  depends_on "glib"
   depends_on "gtk+3"
   depends_on "libarchive"
 
+  on_macos do
+    depends_on "at-spi2-core"
+    depends_on "cairo"
+    depends_on "gdk-pixbuf"
+    depends_on "gettext"
+    depends_on "harfbuzz"
+    depends_on "pango"
```

Comment:
Most of these are from Ventura/Xcode bug.
```suggestion
    depends_on "gettext"
```
https://github.com/Homebrew/homebrew-core/actions/runs/10179435855/job/28155270450?pr=178802#step:4:170
https://github.com/Homebrew/homebrew-core/actions/runs/10179435855/job/28155270943?pr=178802#step:4:173